src/App/BaseURL.php

```diff
@@ -338,12 +338,12 @@ private function determineURLPath()
 		/* Relative script path to the web server root
 		 * Not all of those $_SERVER properties can be present, so we do by inverse priority order
 		 */
-		$relative_script_path = '';
-		$relative_script_path = defaults($this->server, 'REDIRECT_URL', $relative_script_path);
-		$relative_script_path = defaults($this->server, 'REDIRECT_URI', $relative_script_path);
-		$relative_script_path = defaults($this->server, 'REDIRECT_SCRIPT_URL', $relative_script_path);
-		$relative_script_path = defaults($this->server, 'SCRIPT_URL', $relative_script_path);
-		$relative_script_path = defaults($this->server, 'REQUEST_URI', $relative_script_path);
+		$relative_script_path =
+			($this->server['REDIRECT_URL']        ?? '' ) ?:
```

Comment:
I don't really like the spaces in front of a closed bracket. Is this really needed?

Reply:
Nope, nice catch.